ch01.asciidoc

```diff
@@ -191,4 +206,4 @@ Here are some current examples of LN node and wallet applications for different
 | lnd | Server | Full Node | Optional | Full Control |
 | c-lightning | Server | Full Node | Full Node | Full Control |
 | Zap Desktop | Desktop | Full Node | Full Node | Full Control |
-| Eclair | Mobile | Lightweight |
+| Eclair | Mobile | Lightweight | 3rd Party | Full Control |
```

Comment:
So in this table, I think there's a bit more nuance we can capture here, by using the following columns (traits of each wallet):
  * node implementation 
  * device (mobile vs desktop)
  * blockchain data (full node vs SPV)
  * full channel graph storage (full or partial) 
  * local path finding (in device or delegated) 
  * key management (custodial vs non-custodial)
  * channel creation (manual vs autopilot)

Reply:
What do you mean by "node implementation" here? I like this structure a lot and want to explore it in this chapter